packages/react/src/primitives/Tabs/TabsItem.tsx

```diff
@@ -17,23 +17,24 @@ const TabsItemPrimitive: Primitive<TabsItemProps, 'button'> = (
   { className, value, children, onClick, as = 'button', role = 'tab', ...rest },
   ref
 ) => {
-  const { activeTab, setActiveTab } = React.useContext(TabsContext);
+  const { activeTab, setActiveTab, groupID } = React.useContext(TabsContext);
   const isActive = activeTab === value;
   const handleOnClick = (e: React.MouseEvent<HTMLButtonElement>) => {
     if (isTypedFunction(onClick)) {
       onClick?.(e);
     }
     setActiveTab(value);
   };
+  const idValue = value.replace(' ', '-');
```

Comment:
Couple questions:
- Can we just a stable id here as well?
- Any impact if `value` changes between renders?

Reply:
I think we would not want to use a stable id here, because then when switching between tabs a new stable id would be created each time we do that switch. I believe that would lead to the id changing each time we switch off of a tab and back to it. 

I'm not sure about the impact of value changing between renders, but I'm a bit confused on how that could happen. The `value` should only be interacted with in the initial creation of the tab.

Reply:
Does the value of `idValue` in `TabsItem` need to match the `idValue` in the corresponding `TabsPanel`?

Reply:
Yes, because the aria labels are generated based on the idValue, and the tab will point at the panel and vice-versa.

Reply:
Need to ensure that there is a single source of truth rather than evaluate the `idValues` individually, this approach is fragile since an update to `idValue` resolution in one component (but not the other) will break functionality

Reply:
Ok, I think this somewhat goes back to the original question of the user creating tab items and tab panels with accurately matching `value`. The only way for idValue to be different between the `TabItem` and `TabLabel` is if the user assigns mismatching `value` to them. There is also the issue that they _could_ enter the same value for every item and panel in one tab group, which would also break things. Could we just assign numeric `value` to each created `TabItem` and `TabPanel`, and assume that they are defined in the correct order? The current solution was relying on the assumptions that for a given tab group: each tab/panel pair must share the same `value`, and each pair must have a unique `value`.

Reply:
Suggest creating unique per item ids in the Tabs context so there is a single source of truth